build.gradle

```diff
@@ -72,8 +73,14 @@ configurations.all {
 }
 
 dependencies {
+    compile 'io.prometheus:simpleclient:0.6.0'
```

Comment:
All these dependencies should be pulled in from the new version of ms-core 0.4.2-SNAPSHOT